examples/static/microphone-streaming-auto-stop.html

```diff
@@ -34,7 +34,6 @@
 
       var stream = WatsonSpeech.SpeechToText.recognizeMicrophone({
         token: token,
-//        continuous: false, // no longer supported
```

Comment:
@nfriedly actually having looked at this code, I *think* this code already does what you describe. Look at https://github.com/mreinstein/speech-javascript-sdk/blob/master/examples/static/microphone-streaming-auto-stop.html#L40-#L45

Isn't this already the behavior you're referring to?